src/redux/configureStore.js

```diff
@@ -1,12 +1,15 @@
-import { combineReducers, createStore } from 'redux';
-import addRemoveBook from './books/books';
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import addRemoveBook, { fetchBook } from './books/books';
```

Comment:
- [ ] You should use `createAsyncThunk`. Kindly change it. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md#:~:text=Use%20fetch%20or%20axios%20with%20createAsyncThunk%20for%20making%20API%20calls)